solr/cores/conf/schema.xml

```diff
@@ -81,4 +81,18 @@
     </fieldType>
   
    <fieldtype name="ignored" stored="false" indexed="false" multiValued="true" class="solr.StrField" />
+
+    <!-- Multi-valued field for instrument names (e.g., in different languages) -->
+    <field name="instrument_names_s" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
You already have the suffix `_s` defined in your dynamic fields above (it's defined there as a non-multivalued field too). If you want to define a fixed field, you don't need the `_s` suffix.